start-site/src/main/resources/templates/spring-cloud-function-build-setup-azure.mustache

```diff
@@ -0,0 +1,11 @@
+In order to run Spring Cloud Function applications on Microsoft Azure, you can leverage Spring Cloud
+Function Microsoft Azure adapter and {{buildTool}} plugins offered by the cloud platform
+provider.
+
+You will need to add to dependencies and configure the Azure Functions plugin, ensure the
```

Comment:
I don't think we should mention these details here as they're all in the reference doc and we'll automatically get the updates when the reference doc is updated. WDYT?

Reply:
I understood from a comment in a previous review that the Initializr team wanted to see some more details here. However, can change it and provide less details here.

Reply:
Oh, I might've missed that comment. I prefer fewer details here in favor of pointing to the details in the documentation but let's see what the rest of the team thinks.

Reply:
Same here. I think we should focus writing whatever is specific given the actual project in place (like if we detect you've selected A but not B we could add a blurp about the possible combination). Everything else should redirect to other resources as we want that stuff to be available to everyone, not only start.spring.io users.

Reply:
Ok, maybe I have misunderstood the comment. Will change it now.

Reply:
@mbhave @snicoll  done.